### PR TITLE
edits to general_packages.rst

### PR DESCRIPTION
**Pull request by natashawatkins**
I didn't have too many edits

We were thinking about suppressing warnings on the python side - this lecture in particular has a lot of warnings that take up a lot space - what are your thoughts?

### CODE REVIEW

**Comment by jlperla** [comment]
Was this on your desktop or looking at the compiled HTML?  As far as I know, all of the warnings are part of the build-failure @arnavs is working on in #368  although he can confirm.

**Comment by natashawatkins** [comment]
I'm looking at the compare-lectures site

**Comment by jlperla** [comment]
Yes.  @mmcky  and @arnavs  have been trying to see what is wrong with the installation of the machine generating those.  The lecture notes themselves should compile without warning (or close to it) after it is fixed.

File: rst_files/general_packages.rst [hunk 1]
```diff
@@ -15,7 +15,7 @@ Overview
 .. I am not sure I agree... The size of the ecosystem depends on what kind of package you are thinking of
 .. While Julia lacks the massive scientific ecosystem of Python, it has successfully attracted a small army of enthusiastic and talented developers
 
-Julia has both a large number of useful, well written libraries and many incomplete, poorly maintained proofs of concept
+Julia has both a large number of useful, well written libraries and many incomplete poorly maintained proof of concepts
```
Comment (arnavs) [inline rst_files/general_packages.rst]:
```suggestion
Julia has both a large number of useful, well written libraries and many incomplete poorly maintained proofs of concept
```

**Comment by arnavs** [comment]
Thanks @natashawatkins, edits look great. I reverted the one on `proofs of concept => proof of concept` because it's one of those edge cases where it's actually correct the other way (e.g., secretaries-general, courts-martial).

As for the errors, the main one here is QuantEcon.jl's precompilation. We would need to resolve github.com/quantecon/quantecon.jl/issues/286, I think

**Pull request merged**